src/main/scala/org/scalajs/dom/raw/Html.scala

```diff
@@ -3188,6 +3190,12 @@ abstract class HTMLMediaElement extends HTMLElement {
    * MDN
    */
   def audioTracks: AudioTrackList = js.native
+
+  /**
+   * Sets or returns the object which serves as the source of the media associated with the HTMLMediaElement.
+   */
+  var srcObject: js.UndefOr[
+      dom.experimental.mediastream.MediaStream | dom.experimental.mediastream.MediaSource | Blob]
```

Comment:
Consider
```scala
import dom.experimental.mediastream.{MediaStream, MediaSource}
```
at the top so that you can just write
```suggestion
  var srcObject: js.UndefOr[MediaStream | MediaSource | Blob]
```

Reply:
Fixed in https://github.com/scala-js/scala-js-dom/pull/365/commits/053820ea6cc16097476e13349b43bee6fc910f47